From e311dc88d3acac7eca593efea5d95507b02608da Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 13 Apr 2017 12:03:53 +0100
Subject: [PATCH] remove duplicate assignment to critical_temp
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

critical_temp is being assigned to zero twice, remove one of them
as it is redundant.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/thd_zone_cpu.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/thd_zone_cpu.cpp b/src/thd_zone_cpu.cpp
index b6da9bf..051cbf8 100644
--- a/src/thd_zone_cpu.cpp
+++ b/src/thd_zone_cpu.cpp
@@ -51,7 +51,6 @@ cthd_zone_cpu::cthd_zone_cpu(int index, std::string path, int package_id) :
 }
 
 int cthd_zone_cpu::init() {
-	critical_temp = 0;
 	int temp = 0;
 	bool found = false;
 
-- 
2.11.0

